Core/MIPS/IR/IRRegCache.cpp

```diff
@@ -48,8 +48,8 @@ void IRImmRegCache::Discard(IRReg rd) {
 }
 
 IRImmRegCache::IRImmRegCache(IRWriter *ir) : ir_(ir) {
-	memset(&isImm_, 0, sizeof(isImm_));
-	memset(&immVal_, 0, sizeof(immVal_));
+	memset(isImm_, 0, sizeof(isImm_));
+	memset(immVal_, 0, sizeof(immVal_));
```

Comment:
Here's you're doing a change in the right direction, while the one in RegCache.cpp does the opposite. Confused.

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18971/commits/39af23aefa8517e17b9ade1d4c065c47d50901e7